onadata/libs/serializers/merged_xform_serializer.py

```diff
@@ -198,6 +199,7 @@ def create(self, validated_data):
         validated_data['instances_with_geopoints'] = any([
             __.instances_with_geopoints for __ in validated_data.get('xforms')
         ])
+        validated_data['uuid'] = uuid.uuid4().hex
```

Comment:
Could we use the same uuid4 used in https://github.com/onaio/onadata/pull/1773/files#diff-ab8d7875ff4cf8338ac5076d56ff3d47R183? Only that we now use the full hex values rather than the first 6 values?

Reply:
The link seems broken ? Not able to pinpoint the exact line you're referencing

Reply:
Never mind I think the line linked was this ? I'll make the change

```
https://github.com/onaio/onadata/blob/200aefe4a0c8a13f09d6c8e62b79ccb489b55602/onadata/libs/serializers/merged_xform_serializer.py#L182-L183
```